### PR TITLE
fix #4, #5 Describe introduction, installation, quick start and so on

### LINKED ISSUE #4: set up user friendly readme
1. introduction
2. installation

### PR DESCRIPTION
**Pull request by Prodesire**

### CODE REVIEW

File: alibabacloud/ros/README.md [hunk 1]
```diff
@@ -1,20 +1,107 @@
 # ROS OAM Framework
 
-## Install ROS Controller with [Helm v3](https://github.com/helm/helm/releases)
+## Introduction
 
-```
+`ROS OAM Framework` is an implementation of Alibaba Cloud resource orchestration
+that follows OAM standards. It is based on [ROS](https://www.alibabacloud.com/help/doc-detail/28852.html),
+and you can easily orchestrate various service resources of Alibaba Cloud through OAM templates.
```
Comment (wonderflow) [inline alibabacloud/ros/README.md]:
```suggestion
and you can easily orchestrate various service resources of Alibaba Cloud through OAM spec.
```

File: alibabacloud/ros/README.md [hunk 2]
```diff
@@ -90,36 +177,14 @@ spec:
...
-ROS controller will convert appplication configuration to ROS template and the run as a stack.
-```
-kubectl apply -f example/sls
-```
-
-3. delete demo
-ROS controller will delete stack which refers to application configuration.
-```
-kubectl delete applicationconfigurations.core.oam.dev sls-demo
+- Generate workloads which will be generated to current 'workloads' path.
```
Comment (wonderflow) [inline alibabacloud/ros/README.md]:
```suggestion
- Sync workloads will fetch all resource info and generate workloads to current 'workloads' path.
```

File: alibabacloud/ros/README.md [hunk 3]
```diff
@@ -90,36 +177,14 @@ spec:
...
+```shell script
+go run gen.go -i <AccessKeyId> -s <AccessKeySecret>
```
Comment (wonderflow) [inline alibabacloud/ros/README.md:183]: add:

You can apply them again to update this info in cluster.

```
kubectl apply -f workloads/
```

File: alibabacloud/ros/README.md [hunk 4]
```diff
@@ -1,20 +1,107 @@
...
 
-## Check which workload could be used
+## Quick Start
+> Please make sure you have a kubernetes cluster running.
 
-Check the workload list you have:
+After installation and running, let's start with an example which creates 
+Alibaba Cloud [SLS](https://www.alibabacloud.com/help/doc-detail/48869.htm) project, logstore and index.
+
+### Write Templates
```
Comment (wonderflow) [inline alibabacloud/ros/README.md]:
```suggestion
### Write OAM Configurations
```

File: alibabacloud/ros/README.md [hunk 5]
```diff
@@ -1,20 +1,107 @@
...
+In `example/sls`, there are server yaml files which follow OAM standards:
+- `appconf_sls.yaml` is an OAM application which specify three SLS components
+- `comp_sls_project.yaml` is an OAM component which indicates Alibaba Cloud SLS project
+- `comp_sls_logstore.yaml` is an OAM component which indicates Alibaba Cloud SLS logstore
+- `comp_sls_index.yaml` is an OAM component which indicates Alibaba Cloud SLS index
+
+You can refer to these files to write your own OAM template filesto create
```
Comment (wonderflow) [inline alibabacloud/ros/README.md]:
```suggestion
These files will convert to ROS template by this controller and create
```

File: alibabacloud/ros/README.md [hunk 6]
```diff
@@ -1,20 +1,107 @@
...
+In `example/sls`, there are server yaml files which follow OAM standards:
+- `appconf_sls.yaml` is an OAM application which specify three SLS components
+- `comp_sls_project.yaml` is an OAM component which indicates Alibaba Cloud SLS project
+- `comp_sls_logstore.yaml` is an OAM component which indicates Alibaba Cloud SLS logstore
+- `comp_sls_index.yaml` is an OAM component which indicates Alibaba Cloud SLS index
+
+You can refer to these files to write your own OAM template filesto create 
+the Alibaba Cloud resources you want.
+
+### Create Resources by Templates
```
Comment (wonderflow) [inline alibabacloud/ros/README.md]: Change templates to OAM Configurations.

all same in below instructions

**Comment by Prodesire** [comment]
@wonderflow Thanks for review. All fixed!

**Review by wonderflow — APPROVED** [review]

**Pull request merged**